### PR TITLE
[MWES-1679] Drupal on Managed Platform

### PR DESCRIPTION
**Pull request by robpblake**
This PR brings into master all changes required to support Drupal on Managed Platform. 99% of the changes here _do not_ impact the existing Drupal setup. They are in addition to our existing configurations and only applied on Managed Platform.

Most of the managed platform work is contained within the `_docker/drupal/managed-platform` folder. In addition to this there is a `_docker/drupal/dev` folder that supports local development using Managed Platform constructs.

### JIRA Issue Link
* https://issues.jboss.org/browse/MWES-1679

### Verification Process

* The build should go green
* Log into Drupal and ensure there are no dangling configuration changes

### CODE REVIEW

File: _docker/drupal/drupal-filesystem/web/sites/default/settings.php [hunk 1]
```diff
@@ -753,6 +753,5 @@
   include __DIR__ . '/rhd.settings.php';
 }
 
-$settings['install_profile'] = 'lightning';
```
Comment (jordanpagewhite) [inline _docker/drupal/drupal-filesystem/web/sites/default/settings.php]: Was this change intentional @robpblake ?

Comment (robpblake) [reply _docker/drupal/drupal-filesystem/web/sites/default/settings.php]: @jordanpagewhite Yes. This is no longer required for Drupal 8.

cheers,

Rob

Comment (jordanpagewhite) [reply _docker/drupal/drupal-filesystem/web/sites/default/settings.php]: Ah, cool! I wasn't aware of this change. I see the change record now in the 8.3.x branch though. Thanks!
